propagator/xray/opentelemetry-propagator-xray.gemspec

```diff
@@ -28,7 +28,7 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency 'opentelemetry-api', '~> 0.17.0'
 
-  spec.add_development_dependency 'bundler', '>= 1.17'
+  spec.add_development_dependency 'bundler', '~> 2.1.4'
```

Comment:
This seems to be problematic for Ruby 3.0, at least. I think we should remove this change from this PR, and bump the `bundler` version everywhere in the project in a subsequent PR.

Reply:
reverted... sorry for the trouble.